tests/models/semantic_parsing/wikitables/wikitables_decoder_step_test.py

```diff
@@ -163,8 +167,10 @@ def test_get_actions_to_consider_returns_none_if_no_linked_actions(self):
     def test_get_action_embeddings(self):
         action_embeddings = Variable(torch.rand(5, 4))
         self.state.action_embeddings = action_embeddings
+        self.state.output_action_embeddings = action_embeddings
+        self.state.action_biases = Variable(torch.rand(5, 1))
         actions_to_embed = [[0, 4], [1], [2, 3, 4]]
-        embeddings, mask = WikiTablesDecoderStep._get_action_embeddings(self.state, actions_to_embed)
+        embeddings, _, _, mask = WikiTablesDecoderStep._get_action_embeddings(self.state, actions_to_embed)
```

Comment:
You could test the output embeddings here too, right?

Reply:
This whole thing is going to be re-written soon, so I don't want to bother testing this.